api/helpers.js

```diff
@@ -11,16 +11,21 @@ const statusToName = {
   401: 'NotAuthorized'
 };
 
-const reformatDate = (date, oldFormat, newFormat) => moment(date, oldFormat).format(newFormat);
+const reformatDate = (date, oldFormat, newFormat) =>
+  (date instanceof moment ? date : moment(date, oldFormat)).format(newFormat);
```

Comment:
I think this is the job of the calling code really.

Possibly we should have some validation here and throw an error when we get dodgy input.